src/pages/asset.tsx

```diff
@@ -0,0 +1,133 @@
+import OpenInNewIcon from '@mui/icons-material/OpenInNew';
+import { Box, IconButton, Typography, useTheme } from '@mui/material';
+import type { NextPage } from 'next';
+import { useRouter } from 'next/router';
+import { AssetGraphBoxBorrow } from '../components/asset/AssetGraphBoxBorrow';
+import { AssetGraphBoxSupply } from '../components/asset/AssetGraphBoxSupply';
+import { Divider } from '../components/common/Divider';
+import { ReserveDropdown } from '../components/common/ReserveDropdown';
+import { Row } from '../components/common/Row';
+import { Section, SectionSize } from '../components/common/Section';
+import { Skeleton } from '../components/common/Skeleton';
+import { PoolHeader } from '../components/pool/PoolHeader';
+import { useSettings, ViewType } from '../contexts';
+import { usePool, usePoolOracle } from '../hooks/api';
+import { toPercentage } from '../utils/formatter';
+import { getTokenLinkFromReserve } from '../utils/token';
+const Asset: NextPage = () => {
+  const router = useRouter();
+  const theme = useTheme();
+  const { viewType } = useSettings();
+
+  const { poolId, assetId } = router.query;
+  const safePoolId = typeof poolId == 'string' && /^[0-9A-Z]{56}$/.test(poolId) ? poolId : '';
+  const safeAssetId = typeof assetId == 'string' && /^[0-9A-Z]{56}$/.test(assetId) ? assetId : '';
+  const { data: pool } = usePool(safePoolId);
+  const { data: poolOracle, isError: isOracleError } = usePoolOracle(pool);
+  const reserve = pool?.reserves.get(safeAssetId);
+  const hasData = pool && poolOracle && reserve;
+  const link = getTokenLinkFromReserve(reserve);
+
+  return (
+    <>
+      <Row>
+        <Section
+          width={SectionSize.FULL}
+          sx={{ marginTop: '12px', backgroundColor: theme.palette.menu.opaque }}
+        >
+          <PoolHeader name={pool?.config.name ?? 'Unknown'} sx={{ paddingLeft: '12px' }} />
+        </Section>
+      </Row>
```

Comment:
can this be a dropdown?

If we route to the assets page directly with no asset selected, we should just grab the first asset in the `pool.config.reserveList`